lib/factory_bot_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module FactoryBotRails
-  VERSION = "6.4.4".freeze
+  VERSION = "6.4.5".freeze
```

Comment:
```suggestion
  VERSION = "6.4.4".freeze
```

We're going to create a dedicated commit to the version change when this is released.